src/ui/PageHeader.astro

```diff
@@ -18,103 +19,75 @@ const image =
   fallbackImages[Math.floor(Math.random() * fallbackImages.length)];
 
 const smImgUrl = image
-  ? `url(${getImageUrlBuilder(image.image).width(640).height(320).url()})`
+  ? getImageUrlBuilder(image.image).width(640).height(320).url()
   : null;
 const mdImgUrl = image
-  ? `url(${getImageUrlBuilder(image.image).width(1024).height(512).url()})`
+  ? getImageUrlBuilder(image.image).width(1024).height(512).url()
   : null;
 const lgImgUrl = image
-  ? `url(${getImageUrlBuilder(image.image).width(1280).height(640).url()})`
+  ? getImageUrlBuilder(image.image).width(1280).height(640).url()
   : null;
 const xlImgUrl = image
-  ? `url(${getImageUrlBuilder(image.image).width(2560).height(850).url()})`
+  ? getImageUrlBuilder(image.image).width(1400).height(650).url()
   : null;
-const bgPosX = `${(image.image.hotspot?.x ?? 0) * 100}%`;
-const bgPosY = `${(image.image.hotspot?.y ?? 0) * 100}%`;
 ---
 
 <header class={ds.background.color.neutral.defaultLighter}>
-  <div class:list={["sm:mx-auto sm:p-3 sm:relative", "xl:container xl:p-6"]}>
-    <div class:list={["sm:bg-white sm:p-2 sm:shadow-xl", "xl:mx-6", "lg:mt-6"]}>
-      <div class:list={["page-header-bg", "h-40 sm:h-80 xl:h-96"]}></div>
-    </div>
-    <h1
-      class:list={[
-        "overflow-hidden",
-        "-mt-9",
-        "sm:mx-12 sm:absolute sm:bottom-0 sm:left-0",
-        "md:mx-24",
-        // "lg:top-0 lg:mt-8",
-        "xl:max-w-2xl ",
-      ]}
-    >
-      {
-        props.pageTitle.supTitle && (
-          <span class="block">
-            <span
-              class:list={[
-                "font-serif italic",
-                "box-decoration-clone px-3",
-                ds.background.color?.primary.default,
-                ds.typography.color?.primary.contrast,
-              ]}
-            >
-              {props.pageTitle.supTitle}
+  <Container>
+    <div class:list={["p-3 relative", "xl:container xl:p-6"]}>
+        <img
+          srcset={`${smImgUrl} 640w, ${mdImgUrl} 1024w, ${lgImgUrl} 1280w, ${xlImgUrl} 1560w`}
+          sizes="(max-width: 640px) 640px, (max-width: 1024px) 1024px, (max-width: 1280px) 1280px, 1560px"
+          class:list={["inline-block border-8 border-white drop-shadow-md"]}
+        />
+      <h1
+        class:list={[
+          "overflow-visible",
+          "absolute",
+          "-mt-12 sm:-mt-9",
+          "mx-4 sm:mx-12 md:bottom-0 md:left-0",
+          "md:mx-24",
+          "xl:max-w-2xl ",
+        ]}
+      >
```

Comment:
We can massively simplify this now. Let's go with the following:
```suggestion
      <h1 class:list={["mx-3 -mt-8 md:mx-10 md:-mt-14 mb-3 md:mb-6"]}>
```
That should give us a similar effect without the `absolute` positioning